acp/validation.go

```diff
@@ -102,16 +103,27 @@ func ValidateResourceInterface(
 //
 // Learn more about the DefraDB [ACP System](/acp/README.md)
 func validateExpressionOfRequiredPermission(expression string, requiredPermission string) error {
-	exprNoSpace := strings.ReplaceAll(expression, " ", "")
+	exprTrimmed := strings.ReplaceAll(expression, " ", "")
+	if exprTrimmed == acpTypes.RequiredRegistererRelationName {
+		// This is to handle the edge case when it's only owner it won't have `()` around it.
+		return nil
+	}
+
+	fmt.Println("222222222222222222222222222222222222")
+	fmt.Println("expression      : ", expression)
+	fmt.Println("exprTrimmed      : ", exprTrimmed)
+	exprTrimmed = exprTrimmed[1 : len(exprTrimmed)-1] // Trim surrounding ()
+	fmt.Println("exprTrimmed      : ", exprTrimmed)
+	fmt.Println("222222222222222222222222222222222222")
```

Comment:
@Lodek sorry about the `Println`s this was a rough local branch haha, but these small changes should accommodate the sourcehub changes I believe.